MANUAL.md

```diff
@@ -31,11 +32,11 @@ Obtain the source form Github
 Create a virtualenv in the project folder
 
     cd protect
-    virtualenv venv
```

Comment:
This is intentional. The venv in the protect source is venv. In the previous subsection, venvs can have more virtual environments for other tools the user may use